docs/operator-manual/notifications/templates.md

```diff
@@ -20,6 +20,7 @@ Each template has access to the following fields:
 
 - `app` holds the application object.
 - `context` is a user-defined string map and might include any string keys and values.
+- `secrets` provides access to sensitive data stored in `argocd-notifications-secret`
```

Comment:
Would you mind adding an example to docs? I'm not sure it'll be immediately obvious to the user that they're supposed to access secrets via `secrets.<key in data field of secret>`.

Reply:
added a really simple example down below